### ISSUE TITLE
Ring of Magnetizing option to ignore items on Immersive Engineering Conveyor Belts

### ISSUE THREAD

**Issue by CheshireKatt** [Ellpeck/ActuallyAdditions#963]
Basically as the title.  Would it be possible to have the Ring of Magnetizing not pull items on Immersive Engineering conveyor belts like the Ender IO and Botania magnets do?

**Comment by Shadows-of-Fire** [comment]
It already does that; see c09ebe2

**Comment by CheshireKatt** [comment]
Aha, my apologies -- I went back and did some more testing, and yeah, once
they're on the conveyor belt they're properly sticky.  What I thought was
it not working was instead it grabbing the items the instant they either
get placed onto the belt, or leave the belt to be placed into an inventory
(so in both cases they're not actually on the belt at that time).

Specifically, I see it at:

Ejected from Garden Cloche onto conveyor belt
Passes over conveyor with hatch to drop into inventory below the conveyor
belt
Conveyor belt connects directly into an inventory

The Botania ring in this case doesn't grab them in my limited testing, I
think probably because it has a delay before it gets going compared to the
Ring of Magnetization.  The pack I'm playing doesn't have EnderIO right now
(1.12) so I can't compare that one.

I don't know that there's anything really to be done about that (as the
responsiveness of the Ring of Magnetization is a good thing), but I figured
I'd let you know why I thought it wasn't behaving.  :)

Thanks!
